### PR TITLE
feat: reviews section

### PR DESCRIPTION
**Pull request by stefantanasa**

### CODE REVIEW

**Review by Vivy — APPROVED** [review]
100

File: src/component/reviewSection/reviewSection.style.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    &::after {
+      content: '';
+      top: 80px;
+      position: absolute;
+      height: 5px;
+      width: 150px;
+      background-color: #d9b045;
+    }
+  }
+`;
```
Comment (MariaTeo) [inline src/component/reviewSection/reviewSection.style.js:28]: leave an empty line in future for easier readability

File: src/component/reviewSection/reviewSection.style.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  background-color: rgb(247, 242, 226);
+  overflow: hidden;
+  padding: 1.5rem 0;
+  & h1 {
+    padding: 2rem 0;
+    text-align: center;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    &::after {
```
Comment (MariaTeo) [inline src/component/reviewSection/reviewSection.style.js:19]: leave an empty line in future for easier readability

File: src/component/reviewSection/reviewSection.style.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  position: relative;
+  height: 360px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  background-color: rgb(247, 242, 226);
+  overflow: hidden;
+  padding: 1.5rem 0;
+  & h1 {
```
Comment (MariaTeo) [inline src/component/reviewSection/reviewSection.style.js:13]: leave an empty line in future for easier readability

File: src/component/wellcomeMessage/wellcomeMessage.style.js [hunk 4]
```diff
@@ -11,6 +11,7 @@ export const WellcomeMessage = styled.div`
   padding: 1rem 0rem;
   & h1 {
     font-weight: 500;
+    text-align: center;
     b {
```
Comment (MariaTeo) [inline src/component/wellcomeMessage/wellcomeMessage.style.js:15]: leave an empty line in future for easier readability

File: src/component/wellcomeMessage/wellcomeMessage.style.js [hunk 5]
```diff
@@ -11,6 +11,7 @@ export const WellcomeMessage = styled.div`
   padding: 1rem 0rem;
   & h1 {
```
Comment (MariaTeo) [inline src/component/wellcomeMessage/wellcomeMessage.style.js:12]: leave an empty line in future for easier readability

**Review by MariaTeo — APPROVED** [review]
check my comments

**Pull request merged**